package.json

```diff
@@ -64,6 +65,7 @@
     "@types/rbush": "^3.0.3",
     "babel-eslint": "^10.1.0",
     "body-parser": "^1.19.0",
+    "bundle-scss": "^1.5.4",
```

Comment:
only 733 weekly download
Is it well maintained ? If it's not used a lot, it might not be. Is doesn't really inspire trust

Reply:
Looks like https://www.npmjs.com/package/scss-bundle is more wildly used, although not updated since a long time as well